netlify.toml

```diff
@@ -27,7 +27,6 @@ package = "netlify-plugin-checklinks"
 
   # If a link contains these patterns it will not fail the build, but will be reported as a TODO in the build log.
   todoPatterns = [
-    "public/docs/components/central-dash/",
```

Comment:
Why removing this? I am not familiar with this file

Reply:
This line adds the `public/docs/components/central-dash/` to the white list of [checklinks](https://github.com/Munter/netlify-plugin-checklinks). Broken links under those directories are ignored during the CI.
This PR fixes all the broken links under that directory. Therefore we can now remove the directory from the white list so the CI verifies for broken links there too.